lib/dor/release/member_service.rb

```diff
@@ -21,7 +21,7 @@ def sub_collections
       attr_reader :druid
 
       def members_of_type(type)
-        members.filter { |member| member.type == type }
+        members.filter { |member| member.type&.include? type }
```

Comment:
Type of a Cocina::Model object is always a scalar (not an array). See https://github.com/sul-dlss/cocina-models/blob/master/lib/cocina/models/dro.rb#L26 However, it's a URI, so you want to use `collection?` or `dro?` (see https://github.com/sul-dlss/cocina-models/blob/master/lib/cocina/models/checkable.rb#L11)

Reply:
Yeah, it's not -- its returning an array ... I confirmed this by hitting the actual endpoint.  It's basically because it just grabs what comes from solr and sticks in the response (and solr has the object type in a multi-valued field).  

So I guess we are out of sync with the openapi and need to either fix DSA to return a scaler and then revert this change, or adjust openapi to reflect the fact that is an array.

I guess the real question is does it make sense for an object to be of two types.  I don't think so,  yet the solr field we derive this from is indexed in this way.

Reply:
If we just fix DSA, then we don't need this change at all.  No need to revert anything.

Reply:
Yeah sorry, that's what I meant - not do this change in the PR here (though adjusting and keeping the test would be good).

Reply:
I've made https://github.com/sul-dlss/dor-services-app/pull/1060 to fix the DSA problem.

Reply:
undid this change.